exonum/src/crypto.rs

```diff
@@ -529,6 +542,97 @@ impl Default for Hash {
     }
 }
 
+impl CryptoHash for u8 {
+    fn hash(&self) -> Hash {
+        hash(&[*self])
+    }
+}
+
+impl CryptoHash for u16 {
```

Comment:
Do you think it could make sense to use a macro to implement the trait for integers? IMO, it could decrease the amount of copy-pasted code, but could be an overkill.

Reply:
Funnily enough, I made the opposite decision in #443. Quite probably, I'm overly fascinated with meta-programming 😄

Reply:
I'm also fond of metaprogramming, but rust macros disgust me (yeap, even comparing with C++ template metaprogramming). :worried: